sonar-csharp-plugin/pom.xml

```diff
@@ -147,6 +145,46 @@
     </resources>
 
     <plugins>
+      <plugin>
+        <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
+        <artifactId>sonar-packaging-maven-plugin</artifactId>
+        <configuration>
+          <pluginName>SonarC#</pluginName>
```

Comment:
Out of curiosity. What does it change if this property is set in the `properties` section of this `plugin/configuration` section?